src/platform/site-wide/sass/merger.scss

```diff
@@ -259,6 +268,21 @@
             -webkit-font-smoothing: antialiased;
           }
         }
+
+        .va-dropdown-trigger {
+
+          span {
+            max-width: 102px;
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            margin-bottom: -6px;
+
+            @media (max-width: $xsmall-screen) {
+              max-width: 60px;
+            }
+          }
```

Comment:
Ugh, hate seeing this merger file continue to grow. Was it not possible for us to write using mobile-first CSS for this?